README.md

```diff
@@ -46,12 +46,8 @@ If you have Flutter setup locally, on the project's root folder, install pubspec
 
 ### Firebase dependencies
 
-Memo has Firebase dependencies that require a setup before the project can be run locally. The original 
-`GoogleServices-Info.plist` (iOS) and `google-services.json` (Android) are not checked in source control, which means 
-that you must provide your own Firebase project google service files to run the app.
-
-If you want to know how to setup your own Firebase project, check out the 
-[FlutterFire docs](https://firebase.flutter.dev/docs/overview/).
```

Comment:
This was a duplicate of what's written in `ARCHITECTURE.md#firebase`. Updated the docs to reference it